notebooks/Emukit-tutorial-experimental-design-introduction.ipynb

```diff
@@ -0,0 +1,622 @@
+{
```

Comment:
<p>I think it should be "ExpDesign" instead of "Bayesian optimization"</p><p><br></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amzn/emukit/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
changed. Also that paragraph is abit of a lie because we do not have a wrapped loop yet that also instantiates the model. I just used the exp design loop.